### ISSUE TITLE
Decide content for Records tab

### ISSUE THREAD

**Issue by PatReynolds** [FreeUKGen/FreeBMD2#275]
See https://docs.google.com/document/d/1okpxJDQ6fPp1AgT00TewH1HIYr4hVLYhZIkGJEV2J5M/edit?usp=sharing for contents on FreeBMD1 [in progress], FreeCEN and FreeREG.

**Comment by PhilipClarke712** [comment]
I would like to Note that Pat and I exchanged several emails about this quite a few months ago (pre-September) and that included making the "Records" tab. I made a copy of the "About" tab with a view to making a "Records" tab from it- although at the time "Records" was and is still called "Transcriptions". 

I beleive that **all** the 2-ary pages on the spreadsheet that were discussed as being for "Transcriptions" tab are already in Refinery hanging off the current "Transcriptions" tab, which is just a redirect to FreeBMD(1) "Coverage" page; and that has been the case for some time.  But, they don't have all the necessary refinements such, as side navigation panels (nor in-page hyperlinks & skeletal intra-page hyperlinks).

Most of the work on FreeBMD(1) in respect of these pages is done in 3-ary or 4-ary pages, some of which are also there in Refinery in skeletal form. However, a Draft verson of the "Transcriptions" tab is yet to be started, but it is (in a sense) defined by the 3-ary pages that hang off it, or more precisely the place where it should be in Refinery.

My priority is concentrate on the "Help" pages, particularly the missing 3-ary pages. Some of these 3-ary pages are common to both the "Help" tab and the "Transcriptions" tab; and if "Transcriptions" is more relevant place that is where they are being put, with intra-page hyperlinks from the relevant "Help" pages. These pages will progress to MVP state, but not the others.

So in summary, some work is being done on this Issue but it / their priority is obviously lower than "Help" related pages: consistent with this Issue being changed from "documentation" to Production/D&T (Beta)

It is also worth mentioning that some of the 3-ary / 4-ary page currently running on FreeBMD(1) are Perl pages and some of them are the topic of other Issues, such as # 267 and #257.

**Comment by PhilipClarke712** [comment]
I have a draft page on test3 refinery which I hope to have finished soon. There are some unresolved subpages: some need to be coded and some could either be coded or done in html. I will finish the page and indicate the links to such subpages in the form: of [link-text: .... ] [link = .... ] and not make those links (I don't link to draft pages anyway - that is done at sign off).
Then it can be considered if that is what is needed.

**Comment by PatReynolds** [comment]
Hey @PhilipClarke712 I’m using ZenHub in GitHub, [click this link](https://app.zenhub.com/workspaces/freebmd2-5cc6dc330898a02e123026e2/board) to join my workspace and see other features available in GitHub or [download the ZenHub extension](https://www.zenhub.com/extension) and sign up with your GitHub account. 
  _Posted using [ZenHub](https://www.zenhub.com)_

**Comment by PatReynolds** [comment]
Very belated thanks, Philip.  The Records Page is not a refinery page, but is generated from the database and so is coded by a developer (the one on FreeCEN is  a work in progress on the back of the huge amount of work being done on Place within FreeCEN, but the one on FreeREG is as intended.  

On test3, Records = https://test3.freebmd.org.uk/districts  on Beta it is just a link to https://www.freebmd.org.uk/progress.shtml
Other information here could include:
https://www.freebmd.org.uk/statgraphs/

Are you aware of anything else?

The html pages you have made will need to be allocated to Help (the how-tos) or About (the whys and wherefores)

**Comment by PhilipClarke712** [comment]
Sorry about the confusion on my part between what are to implemented  as html pages and what are coded pages. My mind must have been on the intended change of name from Transcripts to Records, and not the content.

**Comment 1.**  I just looked at one record, at the bottom of my screen - **Hammersmith** - which gives its dates of existence and the number of records transcribed. The Dates were "**existing from 1915 to 2000**"; then I looked at the 'bible' that I use: Hammersmith: created 1.7.1913 out of Fulham R.D. Abolished 1.1.2001 and registers transferred to **Hammersmith and Fulham** (which started on 1.1.2001). I then looked at **Fulham** R.D. (on test3)  - **existing from 1875 till 2000**. My 'bible' , written by Brett Langston, which states Fulham was created 1.4.1875 out of Kensington RD'.

 So it looks to me that the dates of existence are being generated from the FreeBMD database itself rather from actual dates. Not a problem, as long as we make that clear. We are not transcribing records any newer than 1991 or 1992 (as appropriate)  so we don't have many newer records, which tend to come from one-name studies - hence we don't yet have a **Hammersmith and Fulham** District on that page.  

**Comment 2.**  Statgraphs is the massive index to every graph of "progress" that is produced every 28 day, during the update cycle:  about 12,500 transcribers have their own progress graph, a progress graph for every quarter, etc. 

Incidentally, two years ago I consistently crashed Refinery, until the size limit was increased, with https://test3.freebmd.org.uk/cms/help-superseded-or-draft/draft-civil-registration-through-the-gro-w-in-p/draft-registration-districts/draft-districts-in-counties-w-i-p/draft-large-table-examples which is just the Registration Districts, in alphabetical order from the counties of Anglesey to Cumberland. Each one of these will have its own progress graph in Statsgraph.

**Comment 3.** There are other measures of "progress", either graphical or numerical, and those are to be found in the Statistics section of https://www.freebmd.org.uk/iindex.shtml which provides a way of focusing in on a family of graphs rather then the omnibus collection of statgraphs. It also gives https://www.freebmd.org.uk/db-latest.shtml which is a **list of transcribers,** as of the last data-base, and their individual-total contribution to date.

**Comment 4.** On FreeBMD(1) we have numerical tables of district-volume and volume-page mappings, and other relationships between volumes, pages and Districts. On FreeBMD(1) https://www.freebmd.org.uk/DistrictInfo.html is a sub-page of Statistics section of https://www.freebmd.org.uk/iindex.shtml#reporting, so we have index pages such as https://www.freebmd.org.uk/district-page-map-index.html which pulls up a Perl page to pull the data off the data pages. Two years ago I did not really know the difference between an html page and and shtml page: so a lot of the shtml pages were put in Refinery as html pages. They probably of no use because they can't pull up Perl pages.

**Comment by PatReynolds** [comment]
Reviewing current iteration (test3) my thoughts are
Currently only showing H - is the plan to have the alphabet along the top of this page, jumping to each letter?

Re @PhilipClarke712 's concern about the source of the content (our database or  another source) - need to add line at the top "This data is derived from the FreeBMD database" (do you think that is ok @PhilipClarke712 - [please suggest something better).

Looking at how "there are no records from this District in these years" is experssed:
![image.png](https://images.zenhubusercontent.com/5b695a5dac3a200155c765e3/70dcd190-f19a-489b-8d69-a8c3d22d5531)
I wonder if it would be better to put 'This District did not exist in this period" rather than "XX" in this slot.
 
I think there need to be back buttons at top, between and bottom

**Comment by PatReynolds** [comment]
Will make a new issue re text changes in my last comment.

**Comment by AlOneill** [comment]
The table `caption`, heard by screen-reader users, is for example, "Information of Hackney District". To sound more natural, the "of" should be replaced with "about".

**Comment by PatReynolds** [comment]
Thanks for this,  @AlOneill I have added that as a new story.

**Comment by AlOneill** [comment]
Also in the District information table, the link text "Click here for more information" will not be meaningful out of context — which is how it could be heard. I suggest, using the Hackney example, "More information about Hackney on UKBMD (opens in new tab)".

**Comment by PhilipClarke712** [comment]
There are two stories on this, and I have made comments on the other story, but I'll summarise them here.

In respect of Alison's comment above, I have suggested what is, in effect, a "small print" paragraph to go under the table because Districts may, and do, get created **during** one of these major periods and also they may be replaced by another District **during** a different major period. So if the search criterion has a single district specified as per Hambledon above - it was abolished on  1st April 1935 and the registers transferred to Surrey South Western (Surrey S W.) - you will not find all the "hits" that you would expect to find.  So, to be sure of finding all the "hits" in the town of Hambledon, in the period 1852 to 1945, as specified above, your search needs to be in **both** Hambledon and Surrey S. W.

In respect of Pat's comment, the volume xx is one of the two Volumes (xx and xxi) allocated to the County of Lancashire in period 1837 to 1851. So a district that did not exist in the period 1837 to 1851, if given the volume number xx, would appear to be a District within the county of Lancashire.

**Comment by DeniseColbert** [comment]
Doc team to look at Philip's comment (plus Vino)

**Comment by DeniseColbert** [comment]
@PatReynolds to send a standard paragraph for below the table. Also needs other text about how to use the page.

**Comment by PatReynolds** [comment]
Having now looked at https://beta.freebmd.org.uk/districts/677/details-of-basildon-district?locale=en I think the best place for the warning is above the link to UKBMD.

Draft: https://docs.google.com/document/d/1-sn0nSbAlQmC1R-shggFu6CmoXo7I8RPlkU2lHrYpuU/edit?usp=sharing

**Comment by richpomfret** [comment]
To now create Epic for 'records' page and build smaller stories for the content required. We can then close.
